[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - LOGS_CAPTURED - Started capturing logs in process (pid: 68907).
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - parse_reps - STEP_START - Started execution of step "parse_reps".
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - parse_reps - Loading file from: /Users/felix/Projects/codenym/DataPipelines/tmp5hel9t2j/storage/house_reps_download using PickledObjectFilesystemIOManager...
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - parse_reps - ASSET_OBSERVATION - DagsterEventType.ASSET_OBSERVATION for step parse_reps
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - parse_reps - LOADED_INPUT - Loaded input "house_reps_download" using input manager "io_manager"
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mDEBUG[0m - __ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - parse_reps - STEP_INPUT - Got input "house_reps_download" of type "Any". (Type check passed).
[32m2023-12-16 07:58:04 -0500[0m - dagster - [34mERROR[0m - [31m__ASSET_JOB - 190996e2-b0a8-4134-86c1-c5ca3807e361 - 68907 - parse_reps - STEP_FAILURE - Execution of step "parse_reps" failed.

dagster._core.errors.DagsterExecutionStepExecutionError: Error occurred while executing op "parse_reps"::

NameError: name 'yaml' is not defined

Stack Trace:
  File "/Users/felix/Projects/codenym/DataPipelines/.venv/lib/python3.11/site-packages/dagster/_core/execution/plan/utils.py", line 54, in op_execution_error_boundary
    yield
  File "/Users/felix/Projects/codenym/DataPipelines/.venv/lib/python3.11/site-packages/dagster/_utils/__init__.py", line 465, in iterate_with_context
    next_output = next(iterator)
                  ^^^^^^^^^^^^^^
  File "/Users/felix/Projects/codenym/DataPipelines/.venv/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 131, in _coerce_op_compute_fn_to_iterator
    result = invoke_compute_fn(
             ^^^^^^^^^^^^^^^^^^
  File "/Users/felix/Projects/codenym/DataPipelines/.venv/lib/python3.11/site-packages/dagster/_core/execution/plan/compute_generator.py", line 125, in invoke_compute_fn
    return fn(context, **args_to_pass) if context_arg_provided else fn(**args_to_pass)
                                                                    ^^^^^^^^^^^^^^^^^^
  File "/Users/felix/Projects/codenym/DataPipelines/datanym/assets/HouseVotes/house_votes.py", line 157, in parse_reps
    raw = yaml.safe_load(f)
          ^^^^
[0m
